apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -466,6 +477,15 @@ export default function CourseOfferingEditor(props) {
           ))}
         </select>
       </label>
+      <label>
+        <div style={styles.flexContainer}>
+          <h3>Published Date </h3>
+          <HelpTip>
+            <p>Select the Published Date of the course offering</p>
+          </HelpTip>
+        </div>
+        <DatePicker date={selectedDate} onChange={handleDateChange} />
```

Comment:
I saw in https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/workshop_filter.jsx#L410 there's a `clearable` option for the DatePicker. Would it be possible to add that here to have a clear button like the video selector? Would this still work with being able to set the date to the blank/cleared option?